lib/livebook_web/live/hooks/app_auth_hook.ex

```diff
@@ -70,6 +78,29 @@ defmodule LivebookWeb.AppAuthHook do
     LivebookWeb.AuthPlug.authenticated?(session, uri.port)
   end
 
+  defp handle_info(:logout, socket) do
+    {:halt, redirect(socket, to: ~p"/logout")}
+  end
+
+  defp handle_info(_event, socket), do: {:cont, socket}
+
+  defp handle_event("logout", %{}, socket) do
+    on_confirm = fn socket ->
+      Phoenix.PubSub.broadcast(Livebook.PubSub, "app", :logout)
```

Comment:
This is so that all tabs get the information, right? In that case I think we can use the same topic in the sidebar hook and here, so that it propagates across both cases. The topic can be very specific, like "logout". Does that make sense?

And if we do that, we can share the logic in the `handle_event` handler between the two places. It can be `SessionHelpers.confirm_logout(socket) -> socket`.